SpheriCart.jl/src/generated_kernels.jl

```diff
@@ -0,0 +1,84 @@
+
+#  This is a generated code for best possible performance with a single input
+
+function _codegen_Zlm(L, T, normalisation) 
+   Flm = generate_Flms(L; normalisation = normalisation)
```

Comment:
Does `generate_Flms` return `Float64` always?

Reply:
thank you - to be fixed.

Reply:
should  also be fixed now. (to be checked and tested)